cpmonitor/admin.py

```diff
@@ -143,6 +116,28 @@ def invitation_link(self, invitation: Invitation):
         )
 
 
+class ChecklistAdmin(ObjectPermissionsModelAdminMixin, admin.ModelAdmin):
```

Comment:
The naming is very similar to model class AdministrationChecklist and could lead into ambiguity (at least I had it 😄). I don't have any suggestions for better naming but maybe you have? 😉

Reply:
Hm no not really. I guess I would like to stick to the convention that those ModelAdmin classes have Admin in the end and try to never use Admin as a shorthand for administration.

Reply:
Ok nevermind 🙂